src/TypeSpec.Extension/Emitter.Csharp/test/TestProjects/array/Generated/tspCodeModel.json

```diff
@@ -26,8 +26,7 @@
       "IsNullable": false
      },
      "IsRequired": true,
-     "IsReadOnly": false,
-     "IsDiscriminator": false
```

Comment:
Any reason we need to remove ' "IsDiscriminator": false' from the tspCodeModel?

Reply:
Discussed with Dapeng on this. I tend to remove unnecessary zero values in the intermediate `tspCodeModel.json`. The benefit is that we have smaller file which makes it easy to read.